include/Ogre_glTF.hpp

```diff
@@ -95,7 +95,7 @@ namespace Ogre_glTF
 		loaderAdapter& operator=(const loaderAdapter&) = delete;
 
 		Ogre::MeshPtr getMesh() const;
-		Ogre::HlmsDatablock* getDatablock() const;
+		Ogre::HlmsDatablock* getDatablock(size_t index) const;
```

Comment:
It would be better not to break the current API (even if it's an internal one if you are just getting the Item or Mesh, not the loaderAdapter directly). So I think we should give it a default value of 0 here.

```suggestion
		Ogre::HlmsDatablock* getDatablock(size_t index = 0) const;
```